### ISSUE TITLE
template overrides

### ISSUE THREAD

**Issue by cadriel** [wobsoriano/vuetify-use-dialog#7]
It would be great if we could override the template in some way.

Specifically we have a requirement to add an icon, and change the color of the card title in the dialog - but there's no exposed property to allow this - nor does Vuetify really expose any props to do with title color. So one option might be to provide a way to provide your own template.

**Comment by wobsoriano** [comment]
For the title color, you can use the `cardTitleProps`:

```ts
const result = await confirm({
  cardTitleProps: {
    class: 'text-indigo-darken-2'
  },
})
```

**Comment by wobsoriano** [comment]
Published v0.5.0. You can now do this:

```ts
import MyDialogTitleComponent from './MyDialogTitleComponent.vue'

const result = await confirm({
  titleComponent: MyDialogTitleComponent,
  titleComponentProps: {},
})
```

**Comment by cadriel** [comment]
looks good, thanks!
